### ISSUE TITLE
sbt client removes empty directories in /tmp

### ISSUE THREAD

**Issue by matil019** [sbt/sbt#6173]
## steps

I'm using the official release of [sbt-1.4.3.tgz](https://github.com/sbt/sbt/releases/download/v1.4.3/sbt-1.4.3.tgz). The OS is Arch Linux.

In an empty directory,

```sh
$ touch build.sbt
$ mkdir /tmp/foo
$ sbt --client shutdown  # or any command will do
[info] entering *experimental* thin client - BEEP WHIRR
[info] server was not detected. starting an instance
[info] terminate the server with `shutdown`
> shutdown
[info] disconnected
$ ls -d /tmp/foo
ls: cannot access '/tmp/foo': No such file or directory
```

Neither `$TMP` nor `$TEMP` are set.

## problem

On launch, `sbt --client` and `sbtn-x86_64-pc-linux` seem to remove directories in `/tmp` as long as they are empty. Non-empty directories and non-readable ones are not affected. The sbt server (without `--client`) doesn't do this.

## expectation

The sbt client shouldn't remove temporary directories unless they are created by sbt itself.

**Comment by eatkins** [comment]
I couldn't reproduce this on Ubuntu 20.04 LTS.

**Comment by eed3si9n** [comment]
@matil019 Thanks for the report. I'm a bit confused by the repro steps:

```
$ touch build.sbt
$ mkdir /tmp/foo
```

Where's your pwd supposed to be?

**Comment by matil019** [comment]
I meant creating a fresh directory and then cd'ing into it.

```sh
$ mkdir ~/test  # or any directory you would like
$ cd ~/test
$ touch build.sbt
$ ...
```

`touch build.sbt` is there because I commented out `-sbt-create` in `/etc/sbt/sbtopts` so that sbt doesn't launch without it. Now, I uncommented it and tried without touching `build.sbt`, but the issue still persists.

[sbt-1.4.4.tgz](https://github.com/sbt/sbt/releases/download/v1.4.4/sbt-1.4.4.tgz) has the same issue (on my PC).

I'll try a fresh Arch Linux installed on VM to see if this is my system-specific or not.
